app/views/spree/shared/_aside.html.erb

```diff
@@ -1,3 +1,3 @@
-<aside>
+<aside class="aside">
```

Comment:
`aside.aside` sounds a bit strange. Maybe we can find a better class name, or just remove the class at all?

Reply:
Being not a best practice to select the HTML element to style it, we should give a class to it. I have to change the class to a better one.

Reply:
Anyway, I left it without class. The class or element name can be defined later when we'll know if keeping this element still makes sense in the current refactor/redesign.